.changeset/real-kangaroos-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hey-api/openapi-ts': minor
```

Comment:
Could you change the release type to patch? Just a preference for how I am currently versioning the package as this is a relatively tiny change